app/views/profiles/edit.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Profiles#edit</h1>
-<p>Find me in app/views/profiles/edit.html.erb</p>
+<h1>Hello <%= @profile.first_name %></h1>
+<%= simple_form_for(@profile) do |f| %>
+    <%= f.input :first_name %>
+    <%= f.input :last_name %>
+    <%= "Email: (not changeable) #{@profile.user.email}" %>
+    <%= f.input :address %>
+    <%# <%= f.input :avatar %>
+    <%#= f.input :avatar_cache. as :hidden %>
+    <%= f.submit "Save", class: "btn btn-primary" %>
```

Comment:
On the DB, we also have a description field on the profil page, with "Text" as a field type